common/config_test.go

```diff
@@ -212,38 +286,25 @@ func TestDownloadableURL_FilePaths(t *testing.T) {
 }
 
 func test_FileExistsLocally(t *testing.T) {
-	if runtime.GOOS == "windows" {
```

Comment:
Again, here, I'd like to keep some of the hardcoded windows-specific tests in addition to your broader ones.

Reply:
...I know this appears crazy. But "C:" is actually valid filename/directory on all the platforms' default filesystems and api. So, `/C:/path` is actually ambiguous. But, packer's prior functionality interprets this as an absolute path, so to retain this prior functionality in packer, this situation needed to be special cased. (Golang also specializes the explicit testing of the drive letter in `os/path_windows.go`)

This is why there's a special handling for it in DownloadableURL (and in the testcase). That special handling essentially ensures that on Windows, this is treated as a separate disk/volume, and that on everything else this is treated as just a directory path. So the FileExistsLocally tests will actually mean different things on each platform. Which makes it hard to use test-fixtures.

But I can attempt and double-up the tests so that both forms are tested on each platform if you guys prefer.

Reply:
...Or are you saying just leave the explicit conditional on line 215 that you had which only runs the `FileExistsLocally` tests on "windows"?

Reply:
That's what I was saying, but you've convinced me your way is the right one. I retract my request for this test.